docs/UserGuide.md

```diff
@@ -264,19 +264,23 @@ Finds students who satisfy the given search criteria.
 
 Format: `find [n/NAME] [s/SCHOOL] [y/YEAR]`
 
-* The search is case-insensitive. e.g `hans` will match `Hans`
 * At least one of the optional fields must be provided.
-* The order of the optional fields do not matter. e.g `n/Hans s/River Valley` will match `s/River Valley n/Hans`
-* Only full words will be matched. e.g `han` will not match `hans`.
-* For the name, students with a name that matches any whole keyword specified for the name will be considered to match for the name.
-* For the school, students with a school that contains any keyword specified for the school will be considered to match for the school.
+* The order of the optional fields do not matter. e.g `n/Hans s/River Valley` is the same as `s/River Valley n/Hans`
+* For the name criteria, only students with a name that matches any full keyword specified will be matched.
```

Comment:
There is also no mention on how name search matches students via OR search

Reply:
Maybe you can clarify what you mean by OR search? Because my understanding is that "matches any full keyword specified" makes the logic quite clear already (and also that is an OR search in that it can match any keyword) without going too deeply into the technicalities

Reply:
oh whoops I must be dyslexic. In that case perhaps you could put "any" in bold and do something for school search?

Reply:
Hi, okay I have bolded `any keyword` if it makes it less confusing. I don't get what you mean by do something for school search but I have bolded `all keywords` in the same spirit!

Reply:
Ye that's what I mean, thanks!